torch_geometric/loader/link_loader.py

```diff
@@ -262,6 +262,7 @@ def filter_fn(
             data.num_sampled_edges = out.num_sampled_edges
 
             data.input_id = out.metadata[0]
+            data.batch_size = out.metadata[0].size(0)
```

Comment:
Is this needed? This is not necessarily correct.

Reply:
I added this line because the test for the DistLinkNeighborLoader was failing because it expected the batch_size field in the batch. I deleted that line. Thank you for your comments.